static/css/blog.css

```diff
@@ -19,7 +19,7 @@ html body .region-inner {
 
 a:link {
   text-decoration:none;
-  color: #2288bb;
+  color: #326DE6;
```

Comment:
It looks like the footer nav links are still blue in the preview: https://deploy-preview-13116--kubernetes-io-master-staging.netlify.com/blog/

I think deleting this would give the desired effect

Reply:
![Screenshot from 2019-03-12 02-14-41](https://user-images.githubusercontent.com/21096774/54178579-b11aef80-446c-11e9-950d-a43d3fd45e29.png)
![Screenshot from 2019-03-12 02-14-46](https://user-images.githubusercontent.com/21096774/54178580-b1b38600-446c-11e9-9ea8-63f113452346.png)

* Netlify current doesn't reflect changes with SASS. Have to run `make sass-develop` to preview changes. 
* So I currently had run this locally. Links are still blue in the content and nav tabs are white, supports hover, visted. (which was a problem previously in [#12842](https://github.com/kubernetes/website/pull/12842#discussion_r260169836)  because just changing color in `line no: 308` in `blog.css` doesn't do the work)

Reply:
Which is obv a betterment compared to the previous commit.

Reply:
hello @Sai-Adarsh. A couple of suggestions if you want all the links to be `white`:
- If you set the color field to `white` or the number representing `white` in line 308 of blog.css, corresponding to:
a.link, a.visited, a.hover, then the blog link will be white
- If you remove the color setting in line 20 of blog.css then all the links in the footer will be white.
nit: there appear to be a number of extraneous commits

Reply:
@kbhawkey Thanks, Tried it, That would be fine for footer nav. But it is seen that it affects the links in the k/blog/ page contents.